src/components/common/NumberOfTransactionsWidget/NumberOfTransactionsWidget.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+
+export const NumberOfTransactionsWidget = ({
+  entriesOnPage,
+  pageEntriesHandler,
+}) => {
+  return (
+    <>
+      <label htmlFor="transactions-select">
+        Select the number of transactions:
+      </label>
+      <select
+        name="transactions"
+        id="transactions-select"
+        value={entriesOnPage}
+        onChange={(event) =>
+          pageEntriesHandler(Number.parseInt(event.target.value))
```

Comment:
if you create list of key-value pair with numbers probably you won't have to parse it

Reply:
If I don't parse the value then it's a string.